ciao-controller/openstack_image.go

```diff
@@ -206,12 +209,50 @@ func getIdentityClient(config Config) (*gophercloud.ServiceClient, error) {
 	return v3client, nil
 }
 
-// Start will get the Image API endpoints from the OpenStack image api,
+// startImageService will get the Image API endpoints from the OpenStack image api,
 // then wrap them in keystone validation. It will then start the https
 // service.
-func Start(config Config) error {
+func (c *controller) startImageService() error {
+
+	dbDir := filepath.Dir(*imageDatastoreLocation)
+	dbFile := filepath.Base(*imageDatastoreLocation)
+	glog.Infof("Image Datastore Location: %v", *imageDatastoreLocation)
+
+	metaDs := &datastore.MetaDs{
+		DbProvider: database.NewBoltDBProvider(),
```

Comment:
can you alias the image datastore to something other than "datastore"? I think it'll be less confusing for future contributors who might get it and the controller's internal/datastore package confused.